doc/source/examples/multiphysics/laser-melting/laser-melting.rst

```diff
@@ -0,0 +1,287 @@
+          set solidus temperature  		= 1700
+    
+          # Specific heat of the liquid phase
+          set specific heat liquid 		= 830
+    
+          # Specific heat of the solid phase
+          set specific heat solid  		= 200
+          
+          set thermal conductivity solid 	= 1.5
+          set thermal conductivity liquid = 33.4
+    
+          # viscosity of the liquid phase
+          set viscosity liquid 		    = 6
+    
+          # viscosity of the solid phase
+          set viscosity solid  		    = 100000
+        end
+    
+      end
+    end
+
+
+.. note:: 
+    Using a ``phase_change`` model for the thermal conductivity, the thermal conductivity of the material varies linearly between ``thermal conductivity solid`` and ``thermal conductivity liquid`` when the temperature is in the range of the solidus and liquidus temperatures.
+
+
+We start the simulation with a rectangular mesh that spans the domain defined by the corner points situated at :math:`[-0.0001, 0]` and
+:math:`[0.0019, 0.0005]`. The first :math:`8,2` couple defines the number of initial grid subdivisions along the length and height of the rectangle. 
+This makes it so our initial mesh is composed of perfect squares. We proceed then to redefine the mesh globally eight times by setting
+``set initial refinement=8``. 
+
+.. code-block:: text
+        
+    #---------------------------------------------------
+    # Mesh
+    #---------------------------------------------------
+    subsection mesh
+            set type = dealii
+            set grid type = subdivided_hyper_rectangle
+            set grid arguments 	= 8, 2 : -0.0001, 0 : 0.0019, 0.000500 : true
+            set initial refinement 	= 8
+    end
+    
+In the ``mesh adaptation subsection``, adaptive mesh refinement is 
+defined for ``temperature``. ``min refinement level`` and ``max refinement level`` are 4 and 8, respectively.
+
+.. code-block:: text
+
+    #---------------------------------------------------
+    # Mesh Adaptation
+    #---------------------------------------------------
+    subsection mesh adaptation
+        set type                    = kelly
+        set variable                = temperature
+        set fraction type           = fraction
+        set max refinement level    = 8
+        set min refinement level    = 4
+        set frequency               = 1
+        set fraction refinement     = 0.5
+        set fraction coarsening     = 0.2
+    end
+
+
+Call the gls_navier_stokes_2d by invoking:  
+
+``mpirun -np 8 gls_navier_stokes_2d laser_melting.prm``
+
+to run the simulation using eight CPU cores. Feel free to use more.
+
+
+.. warning:: 
+    Make sure to compile lethe in `Release` mode and 
+    run in parallel using mpirun. This simulation takes
+    :math:`\approx` 5 mins on 8 processes.
+
+
+
+-------
+Results
+-------
+
+The following image shows the temperature distribution in the simulation domain after 0.0034 seconds.
```

Comment:
It would be interesting to add a bit more of a discussion here. Like analyse the results or something.